esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3703,12 +3669,14 @@ int CWsWuFileHelper::zipAFolder(const char* folder, bool gzip, const char* zipFi
 }
 
 void CWsWuFileHelper::createWUZAPFile(IEspContext& context, Owned<IConstWorkUnit>& cwu, CWsWuZAPInfoReq& request,
-    StringBuffer& zipFileName, StringBuffer& zipFileNameWithPath)
+    StringBuffer& zipFileName, StringBuffer& zipFileNameWithPath, unsigned _getThorSlaveLogThreadPoolSize)
```

Comment:
> _getThorSlaveLogThreadPoolSize

'get' in variable name is misleading.

remove from here and other similar places.

Reply:
ok